src/layout/Footer.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { FaFacebook, FaInstagram, FaTwitter, FaLinkedin } from 'react-icons/fa';
+
+// interface for social media links
+interface SocialLink {
+    name: string;
+    url: string;
+    icon: React.ReactNode;
+}
+
+// interface for Footer props
+interface FooterProps {
+    brandLogo:string;
+    socialLinks: SocialLink[];
+}
+
+const currentYear = new Date().getFullYear();
+
+const Footer: React.FC<FooterProps> = ({ brandLogo, socialLinks }) => {
+    return (
+        <footer className='bg-yellow-500 text-primary py-8'>
+            <div className=''>
+                <div className=''>
+                    {/* Brand section */}
+                    <div>
+                        <img src={brandLogo} alt="Farm2You Logo" className=''/>
```

Comment:
You can get the logo from Figma, import it inside this component and use it here as a source.